vendor/extensions/announcements/app/views/refinery/announcements/show.html.erb

```diff
@@ -1,10 +1,10 @@
 <div class="Announcement">
   <div class="announcement">
     <div class="v2-banner"></div><!-- top green bar -->
-      <div class="announcement__content">
+      <div class="announcement__content container">
         <div class="announcement__content-title"><%= @announcement.title %></div>
```

Comment:
Do these `content-title` and other classes repeat in other show views? It might be worth having a `content` block with elements below it that you reference from each show page and treating this section as its own component if so.

Reply:
Agreed, the Announcement and Report show pages are essentially the same and we could componentize it.

Reply:
This issue is captured here:  #388 and here: #387